sdks/python/apache_beam/runners/interactive/recording_manager.py

```diff
@@ -65,6 +65,13 @@ def var(self):
     """Returns the variable named that defined this PCollection."""
     return self._var
 
+  @property
+  def pcoll(self):
+    # type: () -> beam.pvalue.PCollection
```

Comment:
Will this ever be a `DeferredBase`?

Reply:
No, it shouldn't be. On line interactive_beam.py:469 there's an assertion that the variable is a PCollection before recording it.